README.md

```diff
@@ -195,14 +195,14 @@ In the above example, the first build uses Ubuntu 14.04 (Trusty) which is suppor
 
 #### Additional environment variables
 
-Selected environment variables are passed through to the container. These are currently: `SWIFT_SNAPSHOT`, `KITURA_NIO`, `GCD_ASYNCH` and `TESTDB_NAME`. Additional environment variables can be passed through by setting the `DOCKER_ENVIRONMENT` variable as follows:
+Selected environment variables are passed through to the container. These are currently: `SWIFT_SNAPSHOT` and `KITURA_NIO`. Additional environment variables can be passed through by setting the `DOCKER_ENVIRONMENT` variable as follows:
```

Comment:
Can you show an example of setting `DOCKER_ENVIRONMENT` to a key/value pair?

Reply:
This repo makes use of `DOCKER_ENVIRONMENT`:  https://github.com/IBM-Swift/Swift-SMTP/pull/92

The usage is not a list of key/value pairs as such, it's a list of environment variables to inherit from the current environment. So for example, if I wanted to set a key/value pair within the Docker container I'd use something like:
```
env MYKEY=value DOCKER_IMAGE=ubuntu:16.04 DOCKER_ENVIRONMENT="MYKEY"
```
This defines the `MYKEY` variable in the current environment, then passes it through via `docker run --env`.

Obviously, SWIFT_SNAPSHOT is used pervasively through our repos so there is no benefit to removing it from the default list and requiring people to set DOCKER_ENVIRONMENT.  KITURA_NIO isn't quite so pervasive, so arguably could be removed, though I'd need to update several repos that use it.